chrome-launcher/test/chrome-launcher-test.ts

```diff
@@ -102,4 +102,24 @@ describe('Launcher', () => {
     assert.strictEqual(pid, chromeInstance.pid);
     await chromeInstance.kill();
   });
+
+  it('launching flags without --disable-extensions', async () => {
```

Comment:
maybe something like `it('removes --disable-extensions from flags on enableExtensions', async () => {`?